cpp/abconv/Converter.cc

```diff
@@ -110,7 +110,9 @@ void ab::abconv::Converter::convert() {
 
             // Translate
             auto vtx = ab_result.vertex;
-            evt.shift_position_to(FourVector(vtx.x(), vtx.y(), vtx.z(), vtx.t()));
+            if( ! _afterburner->config().squash_vertex ) {
```

Comment:
Should we call the option "ignore_vertex_position" or "reset_vertex_postion"?

Reply:
"Squash" carries the understanding that there was some finite size to the vertices before it was squashed to (0,0,0,0).  The finite size is necessary for the crabbing effect to be applied.  So I chose this word to hopefully avoid people thinking that this new option also turns off crabbing, which it doesn't.  I'm open to other word choices, but "ignore" and "reset" don't seem as good.

Reply:
"Ignore" and "reset" are means to achieve, the "removal of crabbing" is an effect. We generally prefer to name functionality by its procedure, in case it is reused in a different context.

Reply:
Should squashing disable crabbing (i.e. done pre-afterburner) or to kind of remove crabbing effect on vertex?

Reply:
It will simply remove crabbing effect on vertex.  Particles will still have a momentum modification due to the crabbing, but that is all squashed to (0,0,0,0).  For the lumi pair spectrometer in the far-backward region, this would be a useful option to have.